cpp/Tooling.md

```diff
@@ -2,10 +2,21 @@
 
 ## C++ Compiler
 
+### C++ Standard
+
+The minimal C++ standard officially supported at BBP is C++14, to be reviewed annually based on compiler support.
+An open source project may support a standard as low as C++11 if this is a requirement for its community.
+BBP internal deployment toolchain supports C++17. Therefore software dedicated to BB5 or BlueBrain workstations can pick this standard.
```

Comment:
... and do not require the Intel compilers for optimal performance ...

Reply:
You are right. I will remove the last sentence until we have an intel compiler deployed on BB5 that fully support C++17. Thanks